### DISCUSSION TITLE
Show pinned issues

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#4576] [Mobile]
In the android app there are no pinned issues available, which can be important for some repositories.

**Comment by mxie** [comment]
Thanks for your feedback @Zerotask! I will pass this along to the team for internal discussion.

**Reply by Zerotask** [reply]
Any updates on this?

**Reply by mxie** [reply]
Hi @Zerotask - unfortunately, this is not currently on our roadmap. However, if there's enough demand for it, we will reconsider.

**Comment by hwittenborn** [comment]
I'm +1 on this, I'm really wanting this feature as important issues arrise for my projects when I'm starting to be more and more away from a computer.

**Comment by jamimosas** [comment]
****

**Comment by MiguelRipoll23** [comment]
I usually go to popular repositories like vscode that update their pinned issues monthly and currently this is the reason I’ve to leave the mobile app and go to the website.

The mobile app feels more complete than ever, if this were added it would be much appreciated.
